pydis_site/templates/events/pages/code-jams/8/submissions.html

```diff
@@ -0,0 +1,484 @@
+{% extends "events/base_sidebar.html" %}
+
+{% load static %}
+
+{% block title %}Summer Code Jam 2021{% endblock %}
+
+{% block breadcrumb %}
+    <li><a href="{% url "events:index" %}">Events</a></li>
+    <li><a href="{% url "events:page" path="code-jams" %}">Code Jams</a></li>
+    <li><a href="{% url "events:page" path="code-jams/8" %}">Summer Code Jam 2021</a></li>
+    <li class="is-active"><a href="#">Submissions</a></li>
+{% endblock %}
+
+{% block event_content %}
+    <p>
+        Below is a list of all projects submitted by the end of Summer Code Jam 2021
+    </p>
+
+    <div class="columns is-mobile is-centered">
+        <div class="column is-half">
+            <p class="has-text-centered">Acute Alligators</p>
+        </div>
+        <div class="column is-half">
+            <p class="has-text-centered"><a href="https://github.com/n0remac/Acute-Alligators-2021-Summer-Code-Jam" title="Acute Alligators Repository">GitHub</a></p>
```

Comment:
```suggestion
            <p class="has-text-centered"><a href="https://github.com/n0remac/Acute-Alligators-2021-Summer-Code-Jam" title="Acute Alligators' Repository">GitHub</a></p>
```
The chairs' repository (plural, possesive).

---
This needs to be done for all links

Reply:
I wasn't trying to convey possession but sure, changed in 4625841